### PR TITLE
Derivatives endpoint

### PR DESCRIPTION
**Pull request by will-falconx**
Add `get-derivatives` endpoint to FalconX Python Client

### CODE REVIEW

**Review by james-falconx — CHANGES_REQUESTED** [review]
Looks good overall.  Instead of kwargs, I would explicitly handle the 3 parameters we have.  This fits the current model we have and provides more value for code completion.

File: falconx/client.py [hunk 1]
```diff
@@ -360,6 +360,48 @@ def get_total_balances(self):
         response = self.session.get(self.url + 'balances/total')
         return self._process_response(response)
 
+    def get_derivatives(self, **kwargs):
+        """Get all derivative trade data with current mark-to-market data.
```
Comment (james-falconx) [inline falconx/client.py]: For consistency sake, I would follow the comment blocks of the other functions.
```suggestion
        """
        Get all derivative trade data with current mark-to-market data.
```

Comment (will-falconx) [reply falconx/client.py]: I thought about this too and erroneously went with the Google code style we follow in the other repo - I'll change it

**Review by james-falconx — APPROVED** [review]
LGTM.   Thx for the updates!

**Pull request merged**